src/types/common.rs

```diff
@@ -499,15 +518,37 @@ pub struct Attachments {
 /// A single mailbox.
 #[derive(Debug, Deserialize, XmlSerialize)]
 #[serde(rename_all = "PascalCase")]
-pub struct SingleRecipient {
+pub struct Recipient {
+    #[xml_struct(ns_prefix = "t")]
     pub mailbox: Mailbox,
 }
 
-/// A list of mailboxes.
-#[derive(Debug, Deserialize, XmlSerialize)]
-#[serde(rename_all = "PascalCase")]
-pub struct ArrayOfRecipients {
-    pub mailbox: Vec<Mailbox>,
+/// A util function to deserialize a list of recipients. This is necessary
+/// because quick-xml's serde data model requires the presence of an
+/// intermediate type when dealing with lists, and this is not compatible with
+/// our model for serialization. Note that we could directly deserialize into a
+/// Vec<Mailbox>, which would also simplify this function a bit, but this would
+/// mean using different models to represent a single vs multiple recipient(s).
```

Comment:
todo:
- Use conventional present indicative.
- Separate discussion of usage caveats, design considerations, etc. into a new paragraph. In part, it's easier to tell at a glance which part of the docstring is the short summary; in part, the docs generation tool gives special priority to the first paragraph as a short summary when listing top-level items.
- Use backticks around types. My preference is to also use them around crate names, less sure about the conventionality of that.

```suggestion
/// Deserializes a list of recipients.
///
/// `quick-xml`'s `serde` implementation requires the presence of an
/// intermediate type when dealing with lists, and this is not compatible with
/// our model for serialization.
/// 
/// We could directly deserialize into a `Vec<Mailbox>`, which would also
/// simplify this function a bit, but this would mean using different models
/// to represent single vs. multiple recipient(s).
```